Makefile

```diff
@@ -31,3 +35,9 @@ test:
 # this prevents errors like "No rule to make target 'TestFaultInjection'" when you run "make test TestFaultInjection"
 Test%:
 	@:
+
+image:
+	podman build -t ${CONTAINER_IMAGE} .
+
+push: image
+	podman push ${CONTAINER_IMAGE}
```

Comment:
It will be necessary to run podman login before this will work right?

Reply:
yes... I mean, not if you're already logged in. I added this just so I could build and push to my personal repo, but you can extend it for use in Jenkins, if you want.